packages/frontapp/src/app/components/CategorySearch/index.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import { dimensions, colors } from '@mimir/ui-kit';
+import { AttributesList } from '../FilterAttributes';
+import Button from '../Button';
+const Filters = styled.div`
+  font-weight: 700;
+  font-size: ${dimensions.xl_2};
+  margin-bottom: ${dimensions.base_2};
+`;
+const Title = styled.h3`
+  font-weight: 700;
+  font-size: ${dimensions.base};
+  margin-bottom: ${dimensions.lg};
+`;
+const AttributeWrapper = styled.div`
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(250px, 1fr));
+  row-gap: ${dimensions.xs_2};
+  column-gap: 5rem;
+  width: 100%;
+  height: 100%;
+  margin-bottom: ${dimensions.xl_4};
+`;
+
+const OneCategory = styled.div`
+  display: flex;
+  gap: ${dimensions.xs_2};
+  cursor: pointer;
+  justify-content: space-between;
+  font-size: ${dimensions.base};
+  font-weight: 400;
+`;
+
+const SeeMoreButton = styled.p`
+  color: ${colors.accent_color};
+  text-decoration: underline;
+  cursor: pointer;
+`;
+
+const ButtonWrapper = styled.div`
+  display: flex;
+  justify-content: space-around;
+  gap: ${dimensions.base};
+`;
+
+const StyledCheckBox = styled.input`
+  width: ${dimensions.lg};
+  height: ${dimensions.lg};
+`;
+
+const CategorySearch = () => {
+  return (
+    <>
+      <Filters>Filters</Filters>
+      {AttributesList.map((item) => (
+        <>
+          <Title>{item.Title}</Title>
+          <AttributeWrapper key={item.Title}>
+            {item.Attributes.slice(0, 7).map((attribute) => (
+              <OneCategory>
+                {attribute}
+                <StyledCheckBox type={item.InputType} />
```

Comment:
please write the name of variables or properties of objects with a small letter so that there is a uniform style for the name of variables in our project